src/assets/base-configs/education/Config_CONFIG_ENTITY.json

```diff
@@ -40,20 +40,9 @@
           "link": "/report"
         },
         {
-          "label": "Admin",
```

Comment:
Can you also check all the other `Config_CONFIG_ENTITY` files (for different use cases and languages)?